packages/manager/apps/web/client/app/private-database/database/ram/update/private-database-database-ram-update.controller.js

```diff
@@ -168,8 +147,8 @@ angular.module('App').controller(
         });
     }
 
-    static isProrataDuration({ duration }) {
-      return /^upto/.test(duration);
+    static getCapacityFromPlanCode(planCode) {
```

Comment:
The static keyword is not necessary here
```suggestion
    getCapacityFromPlanCode(planCode) {
```